src/components/common/ColonyHome/ColonyUpgrade/ColonyUpgrade.tsx

```diff
@@ -109,7 +106,8 @@ const ColonyUpgrade = ({ colony }: Props) => {
                       <ExternalLink
                         text={{ id: 'text.learnMore' }}
                         href={getNetworkRelaseLink(
-                          parseInt(colony.version, 10) + 1,
+                          // parseInt(colony.version, 10) + 1,
+                          10,
```

Comment:
I think we have an oversight as in not tracking the version of the colony in the database.

I'll have to add it at some point. (probably when I refactor the ids to avoid forked chain collision)